llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -545,6 +545,16 @@ void RISCVFrameLowering::allocateAndProbeStackForRVV(
       .addReg(SPReg)
       .addReg(TargetReg)
       .setMIFlag(Flag);
+
+  // If we have a dynamic allocation later we need to probe any residuals.
+  MachineBasicBlock *NextMBB = MBBI->getParent()->getSingleSuccessor();
+  if (NextMBB != NULL && NextMBB->begin()->getFlag(MachineInstr::FrameSetup)) {
```

Comment:
Do not use `NULL`. Use `nullptr` or `NextBB && NextMBB->begin()->getFlag(MachineInstr::FrameSetup)`